src/reaper_osara.cpp

```diff
@@ -3503,6 +3512,7 @@ void cmdRemoveOrCopyAreaOfItems(Command* command) {
 				break;
 			}
 			default: {
+				ostringstream s;
```

Comment:
nit: Nothing really wrong with this being here, but we don't need it until later and it tends to make things easier to understand if variables are defined closer to where they're needed.

```suggestion
```